agent/util-scripts/pbench-register-tool

```diff
@@ -175,6 +175,16 @@ while true; do
 		shift
 		_do_test_labels=1
 		;;
+	--persistent)
+		shift
+		oldname=${name}
+		name=$(python3 -c "import sys, json; meta = json.load(open(sys.argv[1])); name = sys.argv[2];  print(name + '-persistent') if not name in meta['persistent'].keys() else print(name)" ${pbench_bin}/tool-scripts/meta.json ${name})
+		;;
+	--transient)
+		shift
+		oldname=${name}
+		name=$(python3 -c "import sys, json; meta = json.load(open(sys.argv[1])); name = sys.argv[2];  print(name + '-transient') if not name in meta['transient'].keys() else print(name)" ${pbench_bin}/tool-scripts/meta.json ${name})
```

Comment:
Seems like you should turn this into a function given the level of obscurity here.